### PR TITLE
ball homework

### PR DESCRIPTION
**Pull request by Funya1337**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.my_class2 {
+  display: none;
+}
+.block-left{width:50%;height:800px;float:left;}
```
Comment (klishevich) [inline css/style.css]: please make right indentation

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      });
+      $(".my_class2").click(function(){
+          $(".my_class1").show();
+          $(".my_class2").hide();
+      });
+    }); // эта закрывающая скобка тоже
+    </script>
+  </head>
+  <body>
+    <center>
```
Comment (klishevich) [inline index.html]: do not use center element

**Review by klishevich — CHANGES_REQUESTED** [review]
please fix

File: index.html (hunk 2 above)
Comment (Funya1337) [reply index.html]: why?

**Pull request merged**